### ISSUE TITLE
Documentation is very hard to follow

### ISSUE THREAD

**Issue by CaptainQuirk** [cosminadrianpopescu/vim-sql-workbench#13]
Hi,

It's very difficult to actually read the documentation. There are many very large pieces of text mixed with commands that are difficult to find and not presented with arguments.

Explanations of how it works is not important in the first place I think. It should be available to those who are interested in it though.

I think there is too much information for a README, too many items are very low level. The readme should focus on the minimal step one has to perform to:
- Open a server (from vim or from the terminal)
- Connect a buffer to the server
- Execute SQL statements
- Explore a Database

The rest should be put in separate wiki pages. (Creating a new database explorer from scratch, Extending the default database explorer)

You should definitely
- create a section with basic mappings to execute sql commands
- make some video recording of how the explorer works, export some extract as a gif to put it right on top of the readme.

It could be great if the readme would include script examples to
- Automatically open a server on a specific port (configurable in vimrc or any .vim file) if the project needs database access. (Something a .projections.json file with vim-projectionist would help doing)
- Automatically connect a buffer of type sql to a server (in a ftplugin/sql.vim file for example)
- Automatically runs the autocomplete command
- Automatically opens the explorer (or several, if you have several db for a project) with a specific profile (could be set in the projections file too)

I'd be glad to help with some of it, but for now, I can't get the plugin working.
 because of a SWServerStart issue.

**Comment by cosminadrianpopescu** [comment]
Yes, you are right. the documentation. However, it is also intended to DBA's who want more from it than just a simple way to send commands to a DBMS. However, you are right that I should move it in some wiki pages and probably I will. 

> It could be great if the readme would include script examples to
> - Automatically open a server on a specific port (configurable in vimrc or any .vim file) if the project needs database access. (Something a .projections.json file with vim-projectionist would help doing)

For this I have to fix first the `SWServerStart`

> - Automatically connect a buffer of type sql to a server (in a ftplugin/sql.vim file for example)

I don't think this is doable so easy, since I need a port on which to connect. One idea would be to add a default port, but then again, you can have several `sqlwbserver` instances opened on different ports. Anyway, it also depends on `SWServerStart`

> - Automatically runs the autocomplete command

Put on top of the buffer a comment like this: `-- before :SWSqlAutocompleteLoad <your-autocomplete-file>`. See the documentation section about the `SQL Buffer`, towards the bottom. 

> - Automatically opens the explorer (or several, if you have several db for a project) with a specific profile (could be set in the projections file too)

Same problem as before. I don't know the port on which `sqlwbserver` is running at least it was started with `SWStartServer`. 

> I'd be glad to help with some of it, but for now, I can't get the plugin working because of a SWServerStart issue.

Try and start it as mentioned in #12 (from the terminal)

**Comment by cosminadrianpopescu** [comment]
If you can test again with the new version (6.0) please let me know if it works. Now the server is not needed. Be careful, that for the new version you need vim 8.0 or higher

**Comment by 4-1-1** [comment]
@cosminadrianpopescu  Hi, Can you upload some example ".vimrc" config?  The docuementation about  start is little hard

**Comment by cosminadrianpopescu** [comment]
```
let g:sw_exe = '/home/lixa/programs/sql-workbench-121/sqlwbconsole.sh'
let g:sw_config_dir = '/home/lixa/.sqlworkbench/'
let g:sw_delete_tmp = 0
let g:sw_save_resultsets = 1
let g:sw_log_to_file = 1
```

This is my sql workbench configuration. I will also add it in the documentation

**Comment by 4-1-1** [comment]
@cosminadrianpopescu ok, thanks . but sqlwbconsole.sh is what ? And can i use it with Mac os, I  have install sql-workbench  by homebrew.

**Comment by 4-1-1** [comment]
I found this . http://www.sql-workbench.net/manual/install.html#shell-using

**Comment by cosminadrianpopescu** [comment]
Read the tutorial from the documentation. This plugin is based on `SQL Workbench/J` software. You have to install that one. You have the links in the tutorial. After you install `SQL Workbench/J` software, you will find in the folder that you installed the `sqlwbconsole.sh` script. You point to that file.
